crates/holochain/src/core/workflow/publish_dht_ops_workflow.rs

```diff
@@ -32,7 +32,7 @@ pub const DEFAULT_RECEIPT_BUNDLE_SIZE: u32 = 5;
 /// Don't publish a DhtOp more than once during this interval.
 /// This allows us to trigger the publish workflow as often as we like, without
 /// flooding the network with spurious publishes.
-pub const MIN_PUBLISH_INTERVAL: time::Duration = time::Duration::from_secs(5);
+pub const MIN_PUBLISH_INTERVAL: time::Duration = time::Duration::from_secs(60);
```

Comment:
I was thinking more on the line of a day than every minute. Or perhaps some kind of exponential backoff?

Reply:
This isn't republish this is just try to publish if you don't have receipts so we can get the receipts

Reply:
I'm aware. I don't think we should try to publish more than once per day, whether or not it's related to gathering receipts, it could still result in way too much network traffic, no?

Reply:
Also this is only at most once a minute if the receipts aren't received (it's triggered from new commits)

Reply:
Seeing as I don't think gathering receipts is that important and should probably be the priority of the republish workflow and not this workflow I tend to agree. I'll change this to once per day at most.